### DISCUSSION TITLE
Does cloudsimplus support the simulation of container scheduling?

### DISCUSSION THREAD

**Discussion by rexxy-sasori** [manoelcampos/cloudsim-plus#418] [Q&A] [answered]
Hi,

New to cloudsimplus here. I guess this may be a beginner question. I see that everything in cloudsimplus is based on VMs. Does this support simulating container scheduling?

Thanks

**Comment by struffel** [comment, marked as answer]
Unfortunately not. CloudSimPlus is based on a fork from CloudSim3, before containers were added to the original CloudSim.
The container implementation in CloudSim4 actually had so many issues that it was one of the main reasons CloudSimPlus was forked in the first place. And so far nobody has re-implemented it (properly) for this version.

See also:
https://github.com/cloudsimplus/cloudsimplus-automation/issues/3
